### PR TITLE
Fix #57: Turn off auto-commit

### LINKED ISSUE #57: Turn off auto-commit
See https://vladmihalcea.com/why-you-should-always-use-hibernate-connection-provider_disables_autocommit-for-resource-local-jpa-transactions/

### PR DESCRIPTION
**Pull request by banterCZ**
See https://vladmihalcea.com/why-you-should-always-use-hibernate-connection-provider_disables_autocommit-for-resource-local-jpa-transactions/

### CODE REVIEW

File: src/main/resources/application.properties [hunk 1]
```diff
@@ -24,8 +24,10 @@ spring.datasource.url=jdbc:postgresql://localhost:5432/powerauth
 spring.datasource.username=powerauth
 spring.datasource.password=
 spring.datasource.driverClassName=org.postgresql.Driver
+spring.datasource.hikari.auto-commit=false
 spring.jpa.properties.hibernate.connection.characterEncoding=utf8
 spring.jpa.properties.hibernate.connection.useUnicode=true
+spring.jpa.properties.hibernate.connection.provider_disables_autocommit=true
```
Comment (romanstrobl) [inline src/main/resources/application.properties:30]: Can we check whether this configuration is duplicate?

See: https://github.com/spring-projects/spring-boot/commit/22de4303c58663e457caa59ea055a9e95078cdb2

I would prefer to configure just the auto-commit on the datasource, if it is enough. But please check if there are any problems with this, it needs to be tested.

**Review by romanstrobl — CHANGES_REQUESTED** [review]
Let's check whether we really need to set both configuration properties.

File: src/main/resources/application.properties (hunk 1 above)
Comment (banterCZ) [reply src/main/resources/application.properties:30]: Thanks, you are correct. Tested, that `HibernateJpaConfiguration` sets `hibernate.connection.provider_disables_autocommit` based on the value of `spring.datasource.hikari.auto-commit`, see https://github.com/spring-projects/spring-boot/blob/2.7.x/spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/orm/jpa/HibernateJpaConfiguration.java#L160

**Review by petrdvorak — APPROVED** [review]
Looks OK

**Pull request merged**